docs/source/faq.rst

```diff
@@ -37,6 +37,18 @@ But this leads to code duplication or an adapter class in your code.
 By using PyVISA as a frontend to many backends, we abstract these things
 from higher level applications.
 
+Pyinstaller can’t find Pyvisa-py, how can I create a standalone executable for distribution to other users?
+-----------------------------------------------------------------------------------------------------------
+
+Pyvisa-py is not technically a valid package name.  Therefore, if you try to import it or if you try to use hiddenimports = [‘pyvisa.py’] in Pyinstaller the executable won’t run. 
```

Comment:
Pyvisa-py is not technically a valid package name.  Therefore, if you try to import it or if you try to use hiddenimports = [‘pyvisa-py’] in Pyinstaller the executable won’t run.